packages/esm-patient-registration-app/src/patient-registration/field/person-attributes/location-person-attribute-field.resource.tsx

```diff
@@ -0,0 +1,54 @@
+import { useCallback, useMemo } from 'react';
+import { type FetchResponse, fhirBaseUrl, openmrsFetch, useDebounce } from '@openmrs/esm-framework';
+import { type LocationEntry, type LocationResponse } from '@openmrs/esm-service-queues-app/src/types';
+import useSWR from 'swr';
+interface IUseLocations {
+  locations: Array<LocationEntry>;
+  isLoading: boolean;
+  loadingNewData: boolean;
+  fetchLocationByUuid: (uuid: string) => void;
+}
+
+export function useLocations(locationTag: string | null, count: number = 0, searchQuery: string = ''): IUseLocations {
```

Comment:
I don't love the `count` here, but even if we keep it, defaulting to 0 seems wrong.

Reply:
`count` is not even being used in the function...